### PR TITLE
CEN-10744: Add recipe for pixel coordinates to wkt conversion

### PR DESCRIPTION
**Pull request by sraavya-CL**
…readme/requirements/inputs

### CODE REVIEW

File: abs_to_wkt/main.ipynb [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "id": "7df70153-489b-4a66-8909-e154b445b8cf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from shapely.geometry import Polygon\n",
```
Comment (gordonwade) [inline abs_to_wkt/main.ipynb:31]: Looks like we're missing an import here: `from PIL import Image`.

File: abs_to_wkt/main.ipynb [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    "# Input your values here:\n",
+    "image_file_path = \"example/input/example_image.jpg\"  # Replace with the actual path to the image file\n",
+    "json_file_path = \"example/input/example_input.json\"  # Replace with the actual path to the JSON file with the pi\n",
+    "\n",
+    "# Extract pixel dimensions of the provided image\n",
+    "image = Image.open(image_file_path)\n",
+    "image_width, image_height = image.size\n",
+    "\n",
+    "# Load image points from the JSON file\n",
+    "image_points = load_points_from_json(json_file_path)\n"
```
Comment (gordonwade) [inline abs_to_wkt/main.ipynb:42]: I'm not sure if `load_points_from_json` is referring to something I'm not seeing. If so, I would add it. Or you can swap for:
```python
with open(json_file_path, 'r') as json_file:
    image_points = json.load(json_file)
```

File: abs_to_wkt/main.ipynb [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+   "execution_count": 36,
+   "id": "07230706-a2de-402c-9cdc-7ef0eccae39b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Function to convert pixel coordinates to a WKT string representing a Polygon\n",
+    "def points_to_wkt_string(image_width: int, image_height: int, pixel_points: list) -> str:\n",
+    "    converted_points = [convert_point(p['x'], p['y'], image_width, image_height) for p in pixel_points]\n",
+    "    polygon = Polygon(converted_points)\n",
+    "    print(wkt_string)\n",
```
Comment (gordonwade) [inline abs_to_wkt/main.ipynb:97]: I think this one was supposed to be at the end.
```suggestion
```

File: abs_to_wkt/main.ipynb [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "POLYGON ((67.36526946107784 36.5625, 50.1497005988024 36.5625, 53.293413173652695 60.15625, 32.18562874251497 60.15625, 67.36526946107784 36.5625))\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Call the function to convert pixel points to WKT string\n",
+    "wkt_string = points_to_wkt_string(image_width, image_height, image_points)"
```
Comment (gordonwade) [inline abs_to_wkt/main.ipynb:125]:
```suggestion
    "wkt_string = points_to_wkt_string(image_width, image_height, image_points)"
    "print(wkt_string)"
```

File: abs_to_wkt/requirements.txt
Comment (gordonwade) [inline abs_to_wkt/requirements.txt:1]: We're missing a `Pillow` dependency here.

File: mask_to_wkt/README.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Mask Extraction
 
-![Mask Extraction and Overlay](example/output/example_mask_overlay.png?raw=false "Mask Extraction and Overlay")
+![Mask Extraction and Overlay](example/output/example_mask_overlay.png?raw=false "Mask Extraction and Overlay")
```
Comment (gordonwade) [inline mask_to_wkt/README.md:3]: Guessing this was an unintentional change?
```suggestion
![Mask Extraction and Overlay](example/output/example_mask_overlay.png?raw=false "Mask Extraction and Overlay")
```

File: abs_to_wkt/main.ipynb
Comment (gordonwade) [inline abs_to_wkt/main.ipynb:1]: This generally looks pretty good! I would reorder the cells slightly so we have function definitions prior to the actual operation. (i.e. the order should be imports, function definitions, load data, perform operations).

**Review by gordonwade — COMMENTED** [review]
Hey @sraavya-CL, this is looking good! Left a few suggestions to help clean things up. When you wrap it up, please make sure that you can run the notebook start to end without any errors as well!

File: abs_to_wkt/main.ipynb (hunk 2 above)
Comment (sraavya-CL) [reply abs_to_wkt/main.ipynb:42]: ah sorry i accidentally deleted that function while cleaning up!

File: mask_to_wkt/README.md (hunk 5 above)
Comment (sraavya-CL) [reply mask_to_wkt/README.md:3]: yep, unintentional

**Review by gordonwade — APPROVED** [review]
Thanks @sraavya-CL - looks great!

**Pull request merged**